carbon_tracker_workflow.ipynb

```diff
@@ -0,0 +1,2510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds_cb = Path("/data/volume_2/EXCITED_prepped_data/CT2022.flux1x1-monthly.nc")
</pre><p>When a variable is named <code>ds_</code> I assume it's an xarray dataset. cb_file would be more clear I think. Or change the functions so that they take an xarray dataset instead of the path.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EXCITED-CO2/excited-workflow/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>